README.md

```diff
@@ -322,6 +323,8 @@ excluded: # paths to ignore during linting. Takes precedence over `included`.
   - Source/ExcludedFolder
   - Source/ExcludedFile.swift
   - Source/*/ExcludedFile.swift # Exclude files with a wildcard
+analyzer_rules: # Rules run by `swiftlint analyze` (experimental)
+  - explicit_self
```

Comment:
how does this play together with the other lists?

Reply:
also this probably should also be mentioned on the list before this snippet

Reply:
Added an explanation